bobtimus/src/bin/fake_bobtimus.rs

```diff
@@ -43,7 +43,7 @@ async fn main() -> Result<()> {
     let cors = warp::cors().allow_any_origin();
 
     let faucet = warp::post()
-        .and(warp::path("faucet"))
+        .and(warp::path!("api" / "faucet" / ..))
```

Comment:
I find it confusing to mix the macro-based filter (`path!`) with normal ones and it seems to require you to do this `..` stuff.

If you want to use the macro, you [should](https://docs.rs/warp/0.2.5/warp/macro.path.html#example) be able to do this:

```rust
warp::path!("api" / "faucet" / Address)
```

Alternatively, you should be able to use two, chained `path()` functions.

Reply:
cool, works.